<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtToolMultiEncrangerMainDialog</class>
 <widget class="QDialog" name="QtToolMultiEncrangerMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiEncranger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/ToolEncrangerArrowDown16x16.png</normaloff>:/images/ToolEncrangerArrowDown16x16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="10">
    <widget class="QPlainTextEdit" name="edit_plaintext">
     <property name="plainText">
      <string>This is plaintext</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="9">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="10">
    <widget class="QPlainTextEdit" name="edit_encrypted_text"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_key">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_arrow_down">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolMultiEncranger.qrc">:/images/ToolMultiEncrangerArrowDown34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="button_encrypt">
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="box_key">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>12345</number>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="button_deencrypt">
     <property name="text">
      <string>Deencrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_arrow_up">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolMultiEncranger.qrc">:/images/ToolMultiEncrangerArrowUp34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ToolMultiEncranger.qrc"/>
 </resources>
 <connections/>
</ui>
